test/artifacts.jl

```diff
@@ -637,7 +637,7 @@ end
             copy_test_package(depot_container, "ArtifactOverrideLoading")
             git_init_and_commit(joinpath(depot_container, "ArtifactOverrideLoading"))
```

Comment:
```suggestion
```

If we are going to `Pkg.develop` the package (instead of `Pkg.add`ing it), there's no need for it to be a Git repo.